### DISCUSSION TITLE
快速收藏点亮收藏图标。

### DISCUSSION THREAD

**Discussion by peace-max** [the1812/Bilibili-Evolved#1313] [功能建议 / Ideas]
最近发现一个新的脚本“bilibili三连”，作用是点击之后能够一键三连，收藏到指定文件夹。可以点亮收藏图标。希望能够对快速收藏，有参考价值
https://greasyfork.org/zh-CN/scripts/372890-bilibili%E4%B8%89%E8%BF%9E

**Comment by the1812** [comment]
嗯嗯 原 issue 那边我看到了的
